config/initializers/redis.rb

```diff
@@ -0,0 +1,3 @@
+require "connection_pool"
+
+RedisPool = ConnectionPool.new(size: 5, timeout: 5) { Redis.new(url: ENV.fetch("REDIS_URL")) }
```

Comment:
This naming confused me a bit. When I saw `RedisPool` in the rest of the code I thought it was a class name.
Should it instead be `Rails.configuration.redis_pool`?